src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/TelemetryChannelEventSource.cs

```diff
@@ -417,6 +417,24 @@ public void LastBackendResponseWhenPutToStorage(int statusCode, double currentDe
             this.WriteEvent(58, statusCode, currentDelayInSeconds, this.ApplicationName);
         }
 
+        [Event(59, Message = "Error dequeuing file: {0}. Exception: {1}.", Level = EventLevel.Error)]
+        public void TransmissionStorageDequeueIOError(string fileName, string exception, string appDomainName = "Incorrect")
```

Comment:
Oops, fixed, I called the wrong message in order to use F12 and forgot to replace it.